### ISSUE TITLE
Delete action in a SelectorViewController

### ISSUE THREAD

**Issue by adrianodiasx93** [xmartlabs/Eureka#1290]
I implemented a custom searchable SelectorViewController with a custom row design for my custom PushRow, and now I would like to be able to also delete the table's rows. What is the best approach to achieve that?

Recently, MultivaluedSection was added and I believe it covers most of the cases, but I am not sure about mine since SelectorViewController use internally a different type of Section (SelectableSection).

In summary, the goal is a row that is editable besides selectable.

Thanks for your time and help! o/

**Comment by adrianodiasx93** [comment]
Also, does MultivaluedSection let you know when and which row was deleted? I have not see any example of it.

In my app, when I tap "delete" I want to send a API request and delete certain data from my local database.

**Comment by adrianodiasx93** [comment]
Even though I am still not able to "swipe it to delete" (I guess it is related to SelectableSection), I can activate the edit mode and delete it by tapping the delete side button. I could do that by replicating the code for remotion from MultivaluedSection in my custom SelectorViewController. 

My question in my previous comment persists though.

**Comment by mats-claassen** [comment]
The question is related to #1276.

Currently the best way to do this in your `SelectorViewController` overriding `rowsHaveBeenRemoved` like this:

```swift
override func rowsHaveBeenRemoved(_ rows: [BaseRow], at indexes: [IndexPath]) {
        super.rowsHaveBeenRemoved(rows, at: indexes)
        print("Rows removed", indexes)
    }
```
